packages/apps/immutable-records/README.md

```diff
@@ -0,0 +1,34 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+
+## Getting Started
+
+First, run the development server:
+
+```bash
+npm run dev
+# or
+yarn dev
+# or
+pnpm dev
+```
+
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+
+You can start editing the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
+
+This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
+
+## Learn More
+
+To learn more about Next.js, take a look at the following resources:
+
+- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
+- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+
+You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+
+## Deploy on Vercel
+
+The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
+
+Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
```

Comment:
Please add this at the top like the other READMEs:

```
<!-- genericHeader start -->

<!-- genericHeader end -->

```

...and run `rush format`.

Reply:
This is not resolved? In general I think comments/issues in PRs should be marked as resolved by the reviewer, not the pull-requester.

Reply:
Right, sorry about that. I assumed it would be pretty straightforward since I added the genericHeader as other apps have it. How is it not resolved yet?

Reply:
Thanks, I did notice format had some additional changes. It actually deletes the description under the `# @kadena/immutable-records` header, not sure if that is intentional. I moved it to below the generic header now.

Reply:
It uses basically this template:

```
# {pkg.name}

{pkg.description}

<img>
```

Definitely a part that's lacking in our docs. I'll see what I can do about that. Thanks for bearing with me!